packages/eth-edge/src/controllers/wallets.controller.ts

```diff
@@ -22,4 +25,11 @@ export class WalletsController {
         const getWalletReq = await useValidate(getWalletsSchema, params);
         return this.walletsService.findAll(getWalletReq.id);
     }
+
+    @Delete("/users/:UserId/deleteWallet")
+    async delete(@Body() body: DeleteWalletRequest): Promise<void> {
+        body.currencyType = CurrencyType.ETHEREUM;
+        const deleteWalletReq = await useValidate(deleteWalletSchema, body);
+        return this.walletsService.delete(deleteWalletReq.address, deleteWalletReq.userId);
```

Comment:
Just pass the entire request obj
```suggestion
        return this.walletsService.delete(deleteWalletReq);
```